.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -52,5 +52,4 @@ Before the changes are marked as `ready-for-merge`:
 - [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
 - [ ] Proper changelog labels are set so that the changelog can be generated automatically
 - [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
-- [ ] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`
-
+- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist and be labeled as `lts-candidate` to be considered
```

Comment:
```suggestion
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).
```